docs/markdown/getting-started.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+This tutorial will help you start an Interledger node locally on you computer and create a local ILP test network.
+
+This tutorial describes how to:
+1. Install [`moneyd`](../concepts/moneyd.md) on your system
+2. Start an [Interledger](https://github.com/interledgerjs/ilp-connector) node on a local test network using moneyd
+3. Send and receive value using the [SPSP](../concepts/spsp.md) (Simple Payment Setup Protocol) API
+
+![Getting-Started-Locally](../images/getting-started-locally.png)
+
+## Before you begin
+* Install a stable version of [Node.js](https://nodejs.org/en/) (10.16.0 LTS is recommended)
```

Comment:
Does it work with the latest version also?

Reply:
10.16.0 LTS is the latest version. It probably works with previous stable versions too but I haven't tried running on them.

Reply:
I think the latest is 12.4.0 -- it's worth checking to make sure it works on that too because many people run the latest (as opposed to the latest stable) version

Reply:
Didn't work with 12.4.0. I get some errors when installing moned.